sdk/resourcemanager/containerservice/armcontainerservice/CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # Release History
 
+## 2.3.0-beta.2 (2022-12-23)
+### Breaking Changes
+
+- Operation `*AgentPoolsClient.AbortLatestOperation` has been changed to LRO, use `*ManagedClustersClient.BeginAbortLatestOperation` instead.
+- Operation `*ManagedClustersClient.AbortLatestOperation` has been changed to LRO, use `*ManagedClustersClient.BeginAbortLatestOperation` instead.
+
+### Features Added
+
+- New const `ManagedClusterPodIdentityProvisioningStateCanceled`
+- New const `ManagedClusterPodIdentityProvisioningStateSucceeded`
+- New const `PrivateEndpointConnectionProvisioningStateCanceled`
+- New const `TrustedAccessRoleBindingProvisioningStateCanceled`
```

Comment:
Could we also make some improvement to this?
Something like:
```
- New value `Enum1A`, `Enum1B` added to type alias `Enum1` # if `Enum1` already exists
- New type alias `Enum2` with values `Enum2A`, `Enum2B`
```

Reply:
used new changelog filter tool update changelog